_core/smart/tutorial-deploy-call.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: smart
+description: "Clarity: Deploy and Interact Tutorial"
+permalink: /:collection/:path.html
+---
+# Deploy and Interact
+
+| Experience | | **Intermediate**  |
+| Duration | | **15 minutes** |
+
+In this tutorial, you learn how to deploy and interact with a smart contract on the Stacks 2.0 blockchain. By the end of this tutorial, you will ...
+
+* Have a Clarity counter smart contract deployed to the Testnet
+* Called public methods of the deployed smart contract
+* Understand how to use the Explorer Sandbox functionality
+* Understand how to use the Blockstack CLI for advanced capabilities
+
+## Overview
+
+* TOC
+{:toc}
+
+{% include note.html content="This tutorial uses a developer preview release of the <a href='https://testnet-explorer.blockstack.org/' target='_blank'>Stacks 2.0 Explorer</a>. Please feel free to report issues or request enhancements on the <a href='https://github.com/blockstack/explorer/issues/new' target='_blank'>blockstack/explorer</a> repository." %}
+
+## Prerequisites
+
+Before you get started, you should complete the [Counter tutorial](tutorial-counter.html). Make sure your `contracts/counter.clar` file looks like this:
+
+```cl
+ (define-data-var counter int 0)
+
+ (define-public (get-counter)
+   (ok (var-get counter)))
+
+ (define-public (increment)
+   (begin
+     (var-set counter (+ (var-get counter) 1))
+     (ok (var-get counter))))
+
+ (define-public (decrement)
+   (begin
+     (var-set counter (- (var-get counter) 1))
+     (ok (var-get counter))))
+```
+
+## Step 1: Accessing the Explorer Sandbox
+
+Open up the <a href="https://testnet-explorer.blockstack.org/sandbox" target="_blank">Stacks 2.0 Explorer Sandbox view</a>. You will be asked to sign in with a Blockstack account. Follow the steps on the screen to complete the process.
+
+Once completed, you will see the Sandbox screen and a confirmation, indicating you were provided with a new STX address for testing purposes.
+
+![new keychain](images/sandbox-new-keychain.png)
+
+## Step 2: Obtaining STX tokens
+
+Uploading and calling smart contracts requires fees to be paid. For this tutorial, we will limit out interaction to the Testnet, which comes with the capability to request STX tokens that can be used to pay the fees ("STX faucet").
```

Comment:
Should clarify that these are fees paid to the network to process the transactions.

Reply:
Done